dev-lang/vala/vala-0.42.3.recipe

```diff
@@ -40,6 +40,7 @@ PROVIDES="
 	cmd:vapigen_$portVers$commandSuffix
 	lib:libvala_$portVers$secondaryArchSuffix = $libVersionCompat
 	lib:libvaladoc_$portVers$secondaryArchSuffix = $libVersionCompat
+	lib:libvalacodegen$secondaryArchSuffix = $portVersion
```

Comment:
What is this? Haikuporter has built this recipe, and hasn't found this lib as provided:
https://build.haiku-os.org/buildmaster/master/x86_64/logviewer.html?buildruns/2486/builds/12519.log

Reply:
There should be at least something providing libvalacodegen, this is working atm but maybe should be changed into devel: instead of lib:?